### PR TITLE
Add image benchmark

### PR DESCRIPTION
**Pull request by Helen-Xie**
ref #25

### CODE REVIEW

**Comment by feiskyer** [comment]
@Helen-Xie Needs another rebase and fix imports.

**Comment by Helen-Xie** [comment]
update @feiskyer

File: pkg/benchmark/image_benchmark.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+			})
+
+			framework.ExpectNoError(err, "failed to list Image: %v", err)
+			Expect(operation.Seconds()).Should(BeNumerically("<", 2), "list Image shouldn't take too long.")
+
+			for _, imageName := range testImageList {
+				imageSpec := &runtimeapi.ImageSpec{
+					Image: imageName,
+				}
+				ic.RemoveImage(imageSpec)
```
Comment (feiskyer) [inline pkg/benchmark/image_benchmark.go]: will those images be kept unremoved if there are errors occurred above? maybe move the removal operation in AfterEach?

Comment (Helen-Xie) [inline pkg/benchmark/image_benchmark.go]: update @feiskyer

**Pull request merged**